components/Carousel/Carousel.js

```diff
@@ -1,6 +1,57 @@
-class Carousel {
+  class Carousel {
+     constructor(element) {
+		  this.element = element;
+      this.carouselitems = element.querySelectorAll(".Carousel__item");
+      this.activeCarouselItem = this.carouselitems[0];
+      this.activeCarouselIndex = 0;
+       this.left = element.querySelector(".Carousel__arrow-left");
+       this.right = element.querySelector(".Carousel__arrow-right");
+       this.left.addEventListener("click", (event) => {
+				 this.updateActive("left");
+				 });
 
-}
+			 this.right.addEventListener("click", (event) => {
+			 this.updateActive("right");
+      });
 
+	}
+
+  updateActive(CarouselArrow) {
+	  if (CarouselArrow === null)
+		 return;
+
```

Comment:
In the updateActive method, lines 20 and 21 can be written together. It makes it easier to find things in the code and keeps it clear and concise.  This may be a matter of personal opinion.  I would have written it like this: 
```
if (CarouselArrow === null) return;
```